docs/openj9_support.md

```diff
@@ -43,18 +43,20 @@ this table over time.
 
 ## Eclipse OpenJ9 releases
 
-| OpenJ9 release  | Release date        | JDK8 (LTS)| JDK11 (LTS) | JDK12   | JDK13    |
-|-----------------|---------------------|-----------|-------------|---------|----------|
-| v0.8.0          | March 2018          | Yes       |             |         |          |
-| v0.9.0          | August 2018         | Yes       |             |         |          |
-| v0.10.0         | September 2018      | No        | Yes(\*2)    |         |          |
-| v0.11.0         | October 2018        | Yes       | Yes         |         |          |
-| v0.12.0         | January 2019        | Yes       | Yes         |         |          |
-| v0.13.0         | March 2019          | No        | No          | Yes(\*2)|          |
-| v0.14.0         | April 2019          | Yes       | Yes         | Yes     |          |
-| v0.15.0         | July 2019           | Yes       | Yes         | Yes     |          |
-| v0.16.0         | September 2019 (\*1)| No        | No          | No      | Yes(\*2) |
-| v0.17.0         | October 2019 (\*1)  | Yes       | Yes         | No      | Yes      |
+| OpenJ9 release  | Release date        | JDK8 (LTS)| JDK11 (LTS) | JDK12    | JDK13     | JDK14     |
+|-----------------|---------------------|-----------|-------------|----------|-----------|-----------|
+| v0.8.0          | March 2018          | Yes       |             |          |           |           |
+| v0.9.0          | August 2018         | Yes       |             |          |           |           |
+| v0.10.0         | September 2018      | No        | Yes (\*2)   |          |           |           |
+| v0.11.0         | October 2018        | Yes       | Yes         |          |           |           |
+| v0.12.0         | January 2019        | Yes       | Yes         |          |           |           |
+| v0.13.0         | March 2019          | No        | No          | Yes (\*2)|           |           |
+| v0.14.0         | April 2019          | Yes       | Yes         | Yes      |           |           |
+| v0.15.0         | July 2019           | Yes       | Yes         | Yes      |           |           |
+| v0.16.0         | September 2019      | No        | No          | No       | Yes (\*2) |           |
+| v0.17.0         | October 2019 (\*1)  | Yes       | Yes         | No       | Yes       |           |
+| v0.18.0         | January 2020 (\*1)  | Yes       | Yes         | No       | Yes       |           |
+| v0.19.0         | March 2020 (\*1)    | Yes       | Yes         | No       | No        | Yes (\*2) |
```

Comment:
JDK8 and 11 should be No on this line. We could remove some of the earlier releases, not sure we need to keep all the history.

Reply:
Good spot. I changed the values and also removed 2018 rows. Will remove JDK 12 when we remove it generally from the docs in October.